### PR TITLE
feat: add images + fix post data + more

### PR DESCRIPTION
**Pull request by WilliamDavidHarrison**
Updated version of #66

### CODE REVIEW

File: src/content/posts/how-does-the-virtual-dom-works.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+title: How does the virtual DOM work in React?
 excerpt: The React Virtual DOM is a crucial concept that enhances...
-createdAt: 25 August 2023
 author: ravisolanki27
+createdAt: August 25, 2023
 ---
 
 The React Virtual DOM is a crucial concept that enhances the performance of React applications by efficiently updating the actual DOM (Document Object Model) when the state of the application changes. It acts as an abstraction layer between the application's logic and the browser's rendering engine. Here's how it works:
 
+![React](../public/images/react.jpg)
```
Comment (WilliamDavidHarrison) [inline src/content/posts/how-does-the-virtual-dom-works.md:11]:
```suggestion
![React](/images/react.jpg)
```

File: src/content/posts/how-does-the-virtual-dom-works.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
+title: How does the virtual DOM work in React?
 excerpt: The React Virtual DOM is a crucial concept that enhances...
-createdAt: 25 August 2023
 author: ravisolanki27
+createdAt: August 25, 2023
 ---
 
 The React Virtual DOM is a crucial concept that enhances the performance of React applications by efficiently updating the actual DOM (Document Object Model) when the state of the application changes. It acts as an abstraction layer between the application's logic and the browser's rendering engine. Here's how it works:
 
+![React](/images/react.jpg)
```
Comment (WilliamDavidHarrison) [inline src/content/posts/how-does-the-virtual-dom-works.md:11]:
```suggestion
![React](../../public/images/react.jpg)

```

File: src/content/posts/how-javascript-works.md [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
 author: abrarlala
+createdAt: July 20, 2023
 ---
 
-# This is how Javascript work 
+# This is how JavaScript works
+
 JavaScript is a versatile programming language commonly used for building interactive and dynamic web applications. It is primarily designed to be run in web browsers but can also be used on the server-side (e.g., with Node.js). Here's a high-level overview of how JavaScript works:
 
+![JS](/images/js.jpg)
```
Comment (WilliamDavidHarrison) [inline src/content/posts/how-javascript-works.md:13]:
```suggestion
![JS](../../public/images/js.jpg)
```

File: src/content/posts/what-is-html.md [hunk 4]
```diff
@@ -2,13 +2,15 @@
...
 excerpt: In this post, we will learn about HTML.
-createdAt: 2023-07-31
 author: iamdiksha
+createdAt: July 31, 2023
 ---
 
 HTML stands for HyperText Markup Language, and it is the standard language used to create and design websites and web pages. It provides the basic structure and content of a web page. HTML uses a system of tags to define the various elements on a web page, such as headings, paragraphs, images, links, etc.
 
-Here's a simple example of an HTML page: 
+![HTML](../public/images/html.jpg)
```
Comment (WilliamDavidHarrison) [inline src/content/posts/what-is-html.md:11]:
```suggestion
![HTML](../../public/images/html.jpg)
```

**Comment by oyepriyansh** [comment]
You can't make changes to someone else's post, and for other changes, please open a new PR.

**Comment by WilliamDavidHarrison** [comment]
> You can't make changes to someone else's post, and for other changes, please open a new PR.

It's me Priyansh

File: src/content/posts/how-does-the-virtual-dom-works.md [hunk 5]
```diff
@@ -1,13 +1,16 @@
...
-createdAt: 25 August 2023
 author: ravisolanki27
+createdAt: August 25, 2023
 ---
 
 The React Virtual DOM is a crucial concept that enhances the performance of React applications by efficiently updating the actual DOM (Document Object Model) when the state of the application changes. It acts as an abstraction layer between the application's logic and the browser's rendering engine. Here's how it works:
 
+![React](../../public/images/react.jpg)
+
+
```
Comment (WilliamDavidHarrison) [inline src/content/posts/how-does-the-virtual-dom-works.md:13]:
```suggestion
```

File: src/content/posts/how-javascript-works.md [hunk 6]
```diff
@@ -1,14 +1,17 @@
...
+createdAt: July 20, 2023
 ---
 
-# This is how Javascript work 
+# This is how JavaScript works
+
 JavaScript is a versatile programming language commonly used for building interactive and dynamic web applications. It is primarily designed to be run in web browsers but can also be used on the server-side (e.g., with Node.js). Here's a high-level overview of how JavaScript works:
 
+![JS](../../public/images/js.jpg)
+
```
Comment (WilliamDavidHarrison) [inline src/content/posts/how-javascript-works.md:14]:
```suggestion
```

File: src/content/posts/what-is-html.md [hunk 7]
```diff
@@ -2,13 +2,15 @@
...
-createdAt: 2023-07-31
 author: iamdiksha
+createdAt: July 31, 2023
 ---
 
 HTML stands for HyperText Markup Language, and it is the standard language used to create and design websites and web pages. It provides the basic structure and content of a web page. HTML uses a system of tags to define the various elements on a web page, such as headings, paragraphs, images, links, etc.
 
-Here's a simple example of an HTML page: 
+![HTML](../../public/images/html.jpg)
+
```
Comment (WilliamDavidHarrison) [inline src/content/posts/what-is-html.md:12]:
```suggestion
```

**Comment by WilliamDavidHarrison** [comment]
> You can't make changes to someone else's post, and for other changes, please open a new PR.

@oyepriyansh I'm just fixing some grammar and stuff

**Comment by oyepriyansh** [comment]
> > You can't make changes to someone else's post, and for other changes, please open a new PR.
> 
> @oyepriyansh I was just fixing some grammar and stuff.

Alright 👌

**Pull request merged**